### PR TITLE
Andrii mruts

### PR DESCRIPTION
**Pull request by cotyhoroshko**

### CODE REVIEW

**Review by BurlyLlama28 — APPROVED** [review]

**Review by irayarka — APPROVED** [review]

File: algo.py [hunk 1]
```diff
@@ -351,7 +355,7 @@ def name() -> str:
 class Task330(TaskWithOneIntValidationParameter):
 
     @staticmethod
-    def _get_deviders(numb):
+    def _get_dividers(numb):
```
Comment (Kavjar) [inline algo.py:358]: It would be better if you made another commit for this change ;)

File: pytest/test_algo.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import pytest
 
 
-# Testing base class with validation method for corrext number answer
+# Testing base class with validation method for correct number answer
```
Comment (Kavjar) [inline pytest/test_algo.py:5]: another commit for comments

**Review by Kavjar — APPROVED** [review]

**Review by ostapbodnar — APPROVED** [review]
